test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -663,3 +663,17 @@ Describe "Console host api tests" -Tag CI {
         }
     }
 }
+
+Describe "Pwsh exe resources tests" -Tag CI {
+    It "Resource strings are embedded in the executable" -Skip:(!$IsWindows) {
+        $pwsh = Get-Item -Path "$PSHOME\pwsh.exe"
+        $pwsh.VersionInfo.FileVersion | Should -BeExactly $PSVersionTable.PSVersion
+        $pwsh.VersionInfo.ProductVersion.Replace("-dirty","") | Should -BeExactly $PSVersionTable.GitCommitId.Replace("v","")
+        $pwsh.VersionInfo.ProductName | Should -BeExactly "PowerShell Core 6"
+    }
+
+    It "Manifest contains compatibility section" -Skip:(!$IsWindows) {
+        $osversion = [System.Environment]::OSVersion.Version
+        $psversiontable.os | Should -Match "$($osversion.Major).$($osversion.Minor)"
```

Comment:
@SteveL-MSFT Please clarify - if we usually use `-BeExactly` for string should we use `-MatchExactly` for string too?

Reply:
Makes sense, will change